src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -159,12 +162,45 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
       messageStore.addMessage(message);
 
-      List<String> messageInformation = new ArrayList<>();
+      List<String> messageInformation = new ArrayList<String>();
       messageInformation.add(user.getName());
       messageInformation.add(conversationTitle);
       messageInformation.add(cleanedMessageContent);
-      Event messageEvent = new Event(UUID.randomUUID(), "Message", message.getCreationTime(), messageInformation);
+      Event messageEvent = 
+          new Event(
+              UUID.randomUUID(), 
+              "Message", 
+              message.getCreationTime(), 
+              messageInformation);
       eventStore.addEvent(messageEvent);
+
+      // Scan the message for "@NemoBot"
+      if (containsExactMatch(cleanedMessageContent, "@NemoBot")) {
```

Comment:
nit: How about we call this `containsWholeWord` since it's not exact match; it ignores case.